src/components/CurrentScenarioSupersetReport/CurrentScenarioSupersetReportHook.js

```diff
@@ -43,23 +43,30 @@ export const useCurrentScenarioSupersetReport = () => {
       scenarios?.map((runner) => ({
         id: runner.id,
         runId: RunnersUtils.getLastRunId(runner),
-      })),
+        lastRun: runner.lastRun,
```

Comment:
In v5 of the Cosmo API, Runner objects don't have a `lastRun` field, but we can probably retrieve the last run data from redux, based on the last run id
We can use one of [these functions](https://github.com/Cosmo-Tech/azure-sample-webapp/blob/abe/Superset-Dynamic-filters-Implementation--PROD-14521/src/state/runner/hooks.js#L84-L90), even though the name of the second one is not really adapted ^^